cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/InMemoryProgramRunnerModule.java

```diff
@@ -111,6 +114,8 @@ protected void configure() {
     // Create webapp http handler factory.
     install(new FactoryModuleBuilder().implement(JarHttpHandler.class, IntactJarHttpHandler.class)
               .build(WebappHttpHandlerFactory.class));
+
+    bind(Store.class).to(DefaultStore.class).in(Scopes.SINGLETON);
```

Comment:
This needs to be fixed to bind ``StoreFactory``, but its causing problems right now, will investigate and fix.